File: backend/src/main/java/com/NBE3_4_2_Team4/global/exceptions/InValidAccessException.java
package com.NBE3_4_2_Team4.global.exceptions;

import lombok.extern.slf4j.Slf4j;

@Slf4j
public class InValidAccessException extends RuntimeException {
    public InValidAccessException(String remoteAddr, String url) {
        super(String.format("someone with ip %s accessed %s in Invalid way", remoteAddr, url));
        log.warn(this.getMessage(), this);
    }
}

File: backend/src/main/java/com/NBE3_4_2_Team4/global/exceptions/InValidAccessException.kt
package com.NBE3_4_2_Team4.global.exceptions;

import com.NBE3_4_2_Team4.global.config.AppConfig.Companion.log

class InValidAccessException(val remoteAddr: String, url: String) :
    RuntimeException("someone with ip $remoteAddr accessed $url in Invalid way") {

    init {
        log.warn(this.message, this)
    }
}
